bench/abc/optimized/abcSweep.ll

```diff
@@ -1936,7 +1936,7 @@ Abc_NodeSetTravIdCurrent.exit:                    ; preds = %tailrecurse, %._cri
 }
 
 ; Function Attrs: nounwind uwtable
-define range(i32 -1, 2) i32 @Abc_NtkCheckConstant_rec(ptr noundef %0) local_unnamed_addr #0 {
+define i32 @Abc_NtkCheckConstant_rec(ptr noundef %0) local_unnamed_addr #0 {
```

Comment:
Seems regression: fails to derive the return range of such a recursive function.

The recursion expression is as follows:
$R^{k+1} = [-1, 2) \cup R^k$
$R^0 = [-1, 2)$
$k$ means the recursion depth, and obviously, the range $R^*$ should have been inferred to $[-1, 2)$